dashboard/app/controllers/peer_reviews_controller.rb

```diff
@@ -14,7 +14,7 @@ def show
   end
 
   def pull_review
-    script = Script.find_by_name(params[:script_id])
+    script = Script.get_from_cache(params[:script_id])
```

Comment:
This is really confusing, that `params[:script_id]` is a script name rather than script ID. Note that `get_from_cache` accepts a name or ID.

TODO(mehal): Fix this upstream.